<?xml version="1.0"?>
<launch>
  <group ns="lwr">
    <arg name="image_left" default="/videostream/left/compressed"/>
    <arg name="image_right" default="/videostream/right/compressed"/>
    <arg name="image_left_info" default="/videostream/left/camera_info" />
    <arg name="image_right_info" default="/videostream/right/camera_info" />


    <rosparam param="/vive/world_offset">[0, 0, 0]</rosparam>
    <rosparam param="/vive/world_yaw">0.0</rosparam>


    <node name="vive_node" pkg="vive_ros" type="vive_node" launch-prefix="$(env HOME)/.steam/ubuntu12_32/steam-runtime/run.sh" output="log" required="true">
      <remap from="~image_left" to="$(arg image_left)" />
      <remap from="~image_right" to="$(arg image_right)" />
      <remap from="~camera_info_left" to="$(arg image_left_info)" />
      <remap from="~camera_info_right" to="$(arg image_right_info)" />
    </node>

    <!--<node type="rviz" name="rviz" pkg="rviz" args="-d $(find vive_ros)/launch/rviz_config.rviz"   launch-prefix="$(env HOME)/.steam/ubuntu12_32/steam-runtime"/>-->

    <node name="left_controller_ps" pkg="vive_ros" type="kuka_posestamped.py" args="controller_LHR_FD7A5BC3 world 30" output="screen"/>
  </group>
</launch>


